ecmascript/parser/src/parser/class_and_fn.rs

```diff
@@ -364,19 +363,18 @@ impl<'a, I: Tokens> Parser<'a, I> {
             }
         }
 
-        self.parse_class_member_with_is_static(accessibility, static_token, decorators)
+        self.parse_class_member_with_is_static(start, accessibility, static_token, decorators)
     }
 
     #[allow(clippy::cognitive_complexity)]
     fn parse_class_member_with_is_static(
         &mut self,
+        start: BytePos,
         accessibility: Option<Accessibility>,
         static_token: Option<Span>,
         decorators: Vec<Decorator>,
     ) -> PResult<'a, ClassMember> {
         let is_static = static_token.is_some();
-        let start = static_token.map(|s| s.lo()).unwrap_or(cur_pos!());
```

Comment:
This start did not include decorators or the accessibility modifier start.

Reply:
You are right. I think I forgot to update it when I added a typescript parser and decorators.

Reply:
Makes sense. Yeah, it's no problem—very easy to fix 🙂